### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by npatilsen**
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/main/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running `generate.ps1`. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version.
	
	**Note: We have recently updated the PSH module called by `generate.ps1` to emit additional data. This would help reduce/eliminate the Code Verification check error. Please run following command**:

	    `dotnet msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules`

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md [hunk 1]
```diff
@@ -285,14 +292,26 @@ await foreach (Page<BasicDigitalTwin> page in asyncPageableResponseWithCharge.As
 
 ### Build ADT Queries
 
-Build an ADT query using an `AdtQueryBuilder`:
+Build an [ADT query](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-query-language) using an `AdtQueryBuilder`. When using a `Where` clause, conditions are separated from the `Where` keyword (see `whereIsOfModel`)
```
Comment (barustum) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md]: Do not include en-us in the link. Reason being is that someone from a different country where locale is not English will still get an English page. If you remove the locale/language in the link, the link will be redirected appropriately to the proper language.

Comment (npatilsen) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md]: good to know!

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md [hunk 2]
```diff
@@ -242,7 +242,10 @@ Query the Azure Digital Twins instance for digital twins using the [Azure Digita
 ```C# Snippet:DigitalTwinsSampleQueryTwins
 // This code snippet demonstrates the simplest way to iterate over the digital twin results, where paging
 // happens under the covers.
-AsyncPageable<BasicDigitalTwin> asyncPageableResponse = client.QueryAsync<BasicDigitalTwin>("SELECT * FROM digitaltwins");
+AsyncPageable<BasicDigitalTwin> asyncPageableResponse = client.QueryAsync<BasicDigitalTwin>(
```
Comment (azabbasi) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md]: You shouldn't manually change the code snippet, you have to add another snippet section in the sample code and reference it here. 
also, we would like to add the method signature with AdtQueryBuilder as the input parameter and not replace the old one with it. users should see both.

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs [hunk 3]
```diff
@@ -288,6 +289,16 @@ await foreach (BasicDigitalTwin twin in asyncPageableResponse)
 
                 #endregion Snippet:DigitalTwinsSampleQueryTwins
 
+                #region Snippet:DigitalTwinsSampleQueryTwinsAdtQueryBuilder
+                // This code snippet demonstrates querying digital twin results using an AdtQueryBuilder, an object that allows for 
+                // fluent-style query construction that makes it easier to write queries.
+                AsyncPageable<BasicDigitalTwin> asyncPageableResponseQueryBuilder = client.QueryAsync<BasicDigitalTwin>(
+                    new AdtQueryBuilder()
+                    .Select("*")
```
Comment (azabbasi) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs]: nit: you can add an indent here at Select

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md [hunk 4]
```diff
@@ -253,14 +253,26 @@ await foreach (BasicDigitalTwin twin in asyncPageableResponse)
...
+```
+
 The SDK also allows you to extract the `query-charge` header from the pageable response. Here's an example of how to query for digital twins and how to iterate over the pageable response to extract the `query-charge` header.
 
 ```C# Snippet:DigitalTwinsSampleQueryTwinsWithQueryCharge
 // This code snippet demonstrates how you could extract the query charges incurred when calling
 // the query API. It iterates over the response pages first to access to the query-charge header,
 // and then the digital twin results within each page.
 
-AsyncPageable<BasicDigitalTwin> asyncPageableResponseWithCharge = client.QueryAsync<BasicDigitalTwin>("SELECT * FROM digitaltwins");
```
Comment (azabbasi) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md]: revert this change

File: sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md [hunk 5]
```diff
@@ -253,14 +253,26 @@ await foreach (BasicDigitalTwin twin in asyncPageableResponse)
 }
 ```
 
+In addition to passing strings as a query parameter, it is possible to pass in an `AdtQueryBuilder` (see following section) object instead of a query in string format.
```
Comment (azabbasi) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md]: I think this belongs to the bottom of the string version api call.

**Review by azabbasi — APPROVED** [review]
